Formula/q.rb

```diff
@@ -1,16 +1,26 @@
+# q - Text as Data
```

Comment:
Could you please remove this line:

```suggestion
```

And squash all the commits into a single one with a message `q 2.0.16`

Reply:
sure (same as the frozen string literals above - `brew audit --strict` failed on this missing comment, this is the reason i've added it in the first place).